src/StructuredLogger/Construction/Construction.cs

```diff
@@ -1082,6 +1082,13 @@ private void AddProperties(TreeNode parent, IEnumerable<KeyValuePair<string, str
                             project.TargetFramework = kvp.Value;
                         }
                     }
+                    // If neither of the above are there - look for the old project system
+                    else if (project.TargetFramework is null && string.Equals(kvp.Key, Strings.TargetFrameworkVersion, StringComparison.OrdinalIgnoreCase))
+                    {
+                        // Note this is untranslted, so e.g. "v4.6.2" instead of "net462" - this is intentional as it

```

Comment:
typo: `unrtanslted` I don't really care much so only fix if you care ;) Otherwise I can fix next time.

Reply:
eh I'm just going to merge